sdk/core/System.Net.ClientModel/api/System.Net.ClientModel.net6.0.cs

```diff
@@ -18,12 +18,12 @@ public partial class ModelReaderWriterOptions
 }
 namespace System.Net.ClientModel.Core
 {
-    public partial interface IJsonModel<out T> : System.Net.ClientModel.Core.IModel<T>
+    public partial interface IJsonModel<out T> : System.Net.ClientModel.Core.IPersistableModel<T>
```

Comment:
Why not make IJsonModel IPersistableJsonModel?

Reply:
I think we were going for shorter names.  The name of the base type doesn't always need to be carried into the derived types.